.esh/commands/locales-full.py

```
@@ -29,200 +29,200 @@
-        langs = langs_arg if langs_arg else all_langs
-        for lang in langs:
-            if lang not in all_langs:
-                print(f"Skip (unknown lang): {lang}")
-                continue
-            merge_cmd(lang, FULL_LOCALES_DIR / f"{lang}.full.yaml")
+	argv = sys.argv[1:]
+	cmd = argv[0] if argv else ""
+	if cmd == "back":
+		langs_arg = argv[1:]  # back 之后的均为语言
+	else:
+		langs_arg = argv if argv else []  # 无 back 则全部为语言（空=全部）
```

Ruff (0.16.0)

[warning] Use ternary operator `langs_arg = argv[1:] if cmd == "back" else argv if argv else []` instead of `if`-`else`-block

(SIM108)